Gemfile

```
@@ -16,6 +16,10 @@ install_if -> { RUBY_PLATFORM !~ /darwin/ } do
   gem "openssl", "= 3.2.0"
 end
 
+install_if -> { RUBY_VERSION < "3.1" } do
+  gem "mixlib-log", ">= 2.0.3", "<= 3.1.1"
+end
```

```suggestion
gem "mixlib-log", ">= 2.0.3", "<= 3.1.1"
```
Just stick with 3.1.1

---

Probably could just put it back in the `.gemspec` even

---

Just did that.